exasol_transformers_extension/udfs/models/base_model_udf.py

```diff
@@ -54,11 +54,24 @@ def get_predictions_from_batch(self, batch_df: pd.DataFrame) -> pd.DataFrame:
         result_df_list = []
         for model_df in \
                 self.extract_unique_model_dataframes_from_batch(batch_df):
-            self.check_cache(model_df)
-            for param_based_model_df in \
-                    self.extract_unique_param_based_dataframes(model_df):
-                result_df = self.get_prediction(param_based_model_df)
-                result_df_list.append(result_df)
+            try:
+                self.check_cache(model_df)
+            except Exception as exc:
+                pass
+                # result_with_error_df = self.get_result_with_error(
+                #     model_df, exc)
+                # result_df_list.append(result_with_error_df)
+            else:
+                for param_based_model_df in \
+                        self.extract_unique_param_based_dataframes(model_df):
+                    try:
+                        result_df = self.get_prediction(param_based_model_df)
+                        result_df_list.append(result_df)
+                    except Exception as exc:
+                        pass
+                        # result_with_error_df = self.get_result_with_error(
+                        #     param_based_model_df, exc)
+                        # result_df_list.append(result_with_error_df)
```

Comment:
maybe extract to function

Reply:
this will be arranged in next round. I already commented out the logic etc.